src/main/antlr4/org/hisp/dhis/parser/expression/antlr/Expression.g4

```diff
@@ -171,6 +171,13 @@ programVariable   // (alphabtical)
 
 numericLiteral
     :   NUMERIC_LITERAL
+    |   INTEGER_LITERAL
+    ;
+
+integerLiteral
+    :   INTEGER_LITERAL
+    |   MINUS? INTEGER_LITERAL
```

Comment:
The question marks after MINUS and PLUS are redundant, and should be removed.

Reply:
You are right
Thanks Jim